### ISSUE TITLE
Create Schedule of Procedures and Assessments

### ISSUE THREAD

**Issue by james-eda** [explore-eda/SilentNight-MT#11]

**Comment by khsarvar** [comment]
Is the schedule of assessments from SAP enough? Do you want us to modify it? If yes, then how?

**Comment by james-eda** [comment]
The original protocol doesn't include both assessments and procedures. Procedures,  or device exposure,  are needed to identify devices,  start times,  end times (and,  eventually,  associate with Audio files)

This data may not be collected on paper but I think its collected as part of the study data.

Imagine the app collects audio to calculate snore.score. The instructions say start app before using device. At first,  it may sync with a snoring device or collect info about it. This way,  we can locate the information from the app log or output and enter it into the crf. This is considered an electronic data transfer, a way to collect study data from external sources. Keep this in mind for the specification and definition documents


________________________________

**Comment by james-eda** [comment]
Consider the records in Trial Visits... would be valuable to see assessments at each of those visits (on site AND off site)
________________________________

**Comment by khsarvar** [comment]
Does that new schedule of assessment goes into new SAP we created?

**Comment by james-eda** [comment]
Yes, as they are planned timepoints for procedures and assessments. I wonder if the inclusion criteria is performed before or after informed consent. I think Informed Consent, first, then Inclusion/Exclusion Criteria, second, but both assessed during the SCREENING VISIT before VISIT 1

**Comment by james-eda** [comment]
Final version should be table in the SAP docx

**Comment by khsarvar** [comment]
What is the format of Schedule of Procedures and Assessments you want me to make?

What about the picture below? Does that work?

![Image](https://github.com/explore-eda/SilentNight-MT/assets/89887135/42c12015-561d-4ef5-b1a8-3befc3db1e0c)

**Comment by james-eda** [comment]
> What is the format of Schedule of Procedures and Assessments you want me to make?
> 
> What about the picture below? Does that work?
> 
> ![Image](https://private-user-images.githubusercontent.com/89887135/298110010-42c12015-561d-4ef5-b1a8-3befc3db1e0c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2ODUxODAsIm5iZiI6MTcwNTY4NDg4MCwicGF0aCI6Ii84OTg4NzEzNS8yOTgxMTAwMTAtNDJjMTIwMTUtNTYxZC00ZWY1LWIxYTgtM2JlZmMzZGIxZTBjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE5VDE3MjEyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwOWU4ODc3ZDlkMWI1NmI1MmNjNjBkYmEwNGJjYWI4MGJkMzE0ZDAwZmVkYThhNjczOTZmYzQ4OWUzNDgyOTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YMEoLCiNfBNLa3hbaNl--hFnNN4xbyLbb4j9BK2MkxA)

Yes - that seems to work. But we may want to introduce another Trial Visit to represent the planned 2 weeks of device use, although the Trial Element can simply have start rules after visits Phone Call1, Phone Call 2, Visit 3.  Hmm... anyways... if we use the format in the picture to create an identical table then that will allow us to add or remove columns and rows as needed.

**Comment by khsarvar** [comment]
Here's identical table that we can use to add or remove columns or rows

https://netorgft5511632.sharepoint.com/:w:/s/edaclinical.com/ER7oWJ2eKrZMp2M9bHC-LekBi0GCPXDXETczg-s-mJMyTA?e=YFHNsK

**Comment by khsarvar** [comment]
Should the rows represent every CRF in order? Every time data is collected from patients?
